src/test/resources/pages/godmode.html

```diff
@@ -56,4 +56,4 @@ <h2>Unpredictable values</h2>
             </ul>
         </div>
     </body>
-</html>
+</html>
```

Comment:
Unrelated change?

Reply:
Left with this...

Reply:
Hey @wkurniawan07 ,

I am puzzled about how to solve this. Is it just an issue of adding "\n"? Plus I didn't change this file at all for my issue, don't know why there is a change here in the first place.

Reply:
> Is it just an issue of adding `\n`?

It is. It's not visible in the "Conversation" view but in the "Files changed" view the diff is displayed like that.